src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs

```diff
@@ -386,29 +367,31 @@ private void HandleSolutionOpened()
             // Go off the UI thread before calling likely expensive call of ExecuteInitScriptsAsync
             // Also, it uses semaphores, do not call it from the UI thread
             Task.Run(async () =>
-            {
-                UpdateWorkingDirectory();
+                {
+                    UpdateWorkingDirectory();
```

Comment:
The spacing looks off here. I don't know if is, just looks weird in the brower.

Reply:
We indent non-first lines of multiline statements:
```
var ... = enumerable.Where(...)
    .Select(...)
    .ToArray(...);
```
It's the same here:
```
Task.Run(() =>
    {
    })
    .ContinueWith(...);
```